src/main/java/com/nutomic/syncthingandroid/activities/LogActivity.java

```diff
@@ -0,0 +1,168 @@
+package com.nutomic.syncthingandroid.activities;
+
+import android.content.Intent;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.support.v7.widget.ShareActionProvider;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.widget.EditText;
+
+import android.support.v4.view.MenuItemCompat;
+import android.widget.ScrollView;
+import android.widget.TextView;
+
+import com.nutomic.syncthingandroid.R;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/**
+ * Shows the log information from Syncthing.
+ */
+public class LogActivity extends SyncthingActivity {
+
+    private final static String TAG = "LogActivity";
+
+    private TextView mLog;
+    private boolean mSyncthingLog = true;
+    private AsyncTask mFetchLogTask;
+    private ScrollView mScrollView;
+    private Intent mShareIntent;
+
+    /**
+     * Initialize Log.
+     */
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.log_activity);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+
+        if (savedInstanceState != null) {
+            mSyncthingLog = savedInstanceState.getBoolean("syncthingLog");
```

Comment:
No, there is no access to the 'item' here and onCreateOptionsMenu is automatically called afterwards, which sets the correct title